packages/bpk-component-modal/src/bpk-modal-dialog.scss

```diff
@@ -99,5 +94,7 @@
 
   &__content {
     padding: $bpk-modal-content-padding;
+    flex-grow: 1;
+    overflow-y: scroll;
```

Comment:
This now causes a scrollbar gutter to appear on desktop:

![screen shot 2018-03-29 at 15 13 16](https://user-images.githubusercontent.com/1440090/38093764-d60c013c-3363-11e8-8707-8fa668250b12.png)

Reply:
Dialogs are also affected:

![image](https://user-images.githubusercontent.com/1440090/38093841-0c54ee8e-3364-11e8-8d5e-f37b37b4e1a1.png)

Reply:
Drawers are also affected:

![image](https://user-images.githubusercontent.com/1440090/38094545-f5a94b10-3365-11e8-98a4-c5752e0e26aa.png)

Reply:
Weird. I don't get that!

Reply:
Ok I do on some browsers...

Reply:
Done - was a 1 line fix